src/wslusc.sh

```diff
@@ -123,9 +123,9 @@ if [[ "$cname" != "" ]]; then
 	fi
 
 	if [[ "$is_gui" == "1" ]]; then
-		winps_exec "Import-Module 'C:\\WINDOWS\\system32\\WindowsPowerShell\\v1.0\\Modules\\Microsoft.PowerShell.Utility\\Microsoft.PowerShell.Utility.psd1';\$s=(New-Object -COM WScript.Shell).CreateShortcut('$tpath\\$new_cname.lnk');\$s.TargetPath='C:\\Windows\\System32\\wscript.exe';\$s.Arguments='$script_location_win\\runHidden.vbs $distro_location_win $distro_param \"cd ~; export DISPLAY=:0; $customenv $cname\"';\$s.IconLocation='$iconpath';\$s.Save();"
+		winps_exec "Import-Module 'C:\\WINDOWS\\system32\\WindowsPowerShell\\v1.0\\Modules\\Microsoft.PowerShell.Utility\\Microsoft.PowerShell.Utility.psd1';\$s=(New-Object -COM WScript.Shell).CreateShortcut('$tpath\\$new_cname.lnk');\$s.TargetPath='C:\\Windows\\System32\\wscript.exe';\$s.Arguments='$script_location_win\\runHidden.vbs \"$distro_location_win\" $distro_param \"cd ~; export DISPLAY=:0; $customenv $cname\"';\$s.IconLocation='$iconpath';\$s.Save();"
```

Comment:
Change \"cd ~; export DISPLAY=:0;  by \"bash -l -c 

So the user environment will be loaded including DISPLAY for WSL2 and XServer in pengwin

Reply:
in this way, it will brake wslu in other distros. I will look into this.

Reply:
@crramirez I will possibly do that in 2.3.4. I will need to release 2.3.3 to fix the username problem first.

Reply:
@crramirez I realized what you mean after seeing your code. I have added the change to the `wslusc` to reduce repetitive files, and also made some changes to avoid the possible issues. Could you test it? Thanks!